deps.bzl

```diff
@@ -164,9 +164,16 @@ def daml_deps():
     if "io_grpc_grpc_java" not in native.existing_rules():
         http_archive(
             name = "io_grpc_grpc_java",
-            strip_prefix = "grpc-java-1.19.0",
-            urls = ["https://github.com/grpc/grpc-java/archive/v1.19.0.tar.gz"],
-            sha256 = "81d1e12bf0f8bd1560eed7c75f24d8bb8e7368dcf07802586e439c85cf89b005",
+            strip_prefix = "grpc-java-7e5e6e604a9a74cb62a0aa6edbdb47f2ae676ed7",
+            # This corresponds to the v1.21.x branch which is slightly ahead of the latest release
+            # in that series and has a fix for using custom Java toolchains.
+            urls = ["https://github.com/grpc/grpc-java/archive/7e5e6e604a9a74cb62a0aa6edbdb47f2ae676ed7.tar.gz"],
+            sha256 = "92dea47bdb47929465968b564238f9f8d8fcd28765e4c446380a819ca6fb5208",
+            # The java_grpc_library rules do not produce a "java" provider
```

Comment:
Is the issue a missing `xyz.java` or a missing `xyz[Java]`? I think the former is being deprecated, maybe the latter is present?

Reply:
It complains about the former. It looks like it provides a `JavaInfo` provider so I guess `bazel_common` needs to switch over their `javadoc_library` to use that.